libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/NodeVerifier.java

```diff
@@ -0,0 +1,7 @@
+package com.mapbox.services.android.navigation.ui.v5.instruction;
+
+import com.mapbox.api.directions.v5.models.BannerComponents;
+
+abstract class NodeVerifier {
```

Comment:
I'm curious, why using an `abstract class` instead of an `interface`?

Reply:
Oops, this was outdated from when there was other logic in this class.